sdk/core/ts-http-runtime/review/azure-core-comparison.diff

```diff
@@ -1424,6 +1424,15 @@ index 3df6036..d653f06 100644
  import type { PathUncheckedResponse } from "./common.js";
  
  /**
+@@ -22,7 +23,7 @@ export function createRestError(
+   const message =
+     typeof messageOrResponse === "string"
+       ? messageOrResponse
+-      : (internalError?.message ?? `Unexpected status code: ${resp.status}`);
```

Comment:
Would you do me a favor and update the unbranded Core here too so they don't diverge? Soon the unbranded version will be what we're using so don't want to risk losing this change by accident

Reply:
ah, completely missed that. I pushed this change in https://github.com/Azure/azure-sdk-for-js/pull/33623/commits/20b1d315de4cc53981f94eb6caf9826717a8dd00

Reply:
Thanks!